build.gradle

```diff
@@ -46,6 +46,16 @@ dependencies {
     implementation('org.springframework.retry:spring-retry')
     implementation('org.springframework:spring-aspects')
 
+
+    // Override the default version of elasticSearch, we're still using V6
+    def elasticSearchVersion = '6.8.3'
+    implementation('org.elasticsearch:elasticsearch:' + elasticSearchVersion)
```

Comment:
syntax is normally: `implementation("org.elasticsearch:elasticsearch:${elasticSearchVersion}")`

Reply:
Done